frontend/src/app/patients/EditPatient.tsx

```diff
@@ -241,7 +245,9 @@ const EditPatient = (props: Props) => {
                       disabled={editPersonLoading || !formChanged}
                       onClick={onSave}
                     >
-                      {editPersonLoading ? "Saving..." : "Save changes"}
+                      {editPersonLoading
+                        ? "Saving..."
```

Comment:
I guess "Saving..." is untranslated as well

Reply:
Sure is... @Rebecca-LM _this_ is surely the last one 😁

Reply:
@nathancrtr @nickscialli-usds “Save changes” = “Guardar cambios”, “Saving” = “Guardando”